src/main/java/seedu/workbook/logic/commands/FindCommand.java

```diff
@@ -24,9 +26,9 @@ public class FindCommand extends Command {
             + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
would be good to edit the `MESSAGE_USAGE` which is shown to users as well, so that they understand they have to specify prefixes